пересдача/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace пересдача
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var count = 0;
+
+            for (int i = 1; i <= 1000; i++)
```

Comment:
Но нужно было вычислить _f_(10^6):
```suggestion
            for (int i = 1; i <= 1000000; i++)
```